pb/custom_evaluator/custom_evaluator.proto

```diff
@@ -52,6 +52,12 @@ message TestResultComparison {
     //
     // NOTE: Only supported for string and numerical values
     string test_result_error_field = 4;
+
+    // This will find the scalar field with the given name within the TargetResult
+    // proto. The value of that field will be used to evaluate.
+    //
+    // NOTE: Only supported for TestStatus values.
+    string target_result_field = 5;
```

Comment:
Is this the best way to configure a feature like this? Why not a boolean for "target_status"? What else would a user want to set it to?

Reply:
+1, the other fields are intended to be more generic but since this is using status specifically, I think you can be more specific about what to compare.

Reply:
Currently only target status is used, looks like we don't have plans to add other fields, I'll change it to `bool`.